### DISCUSSION TITLE
Visual Regression Test for simple .png files

### DISCUSSION THREAD

**Discussion by dmaulick** [percy/cli#939] [Support] [answered]
Hi, I simply want to visual regression test plain old .png files by passing them to the percy cli.
I know this is not the typical usage but I am trying to leverage percy to do visual regression tests on mobile. As a result I am taking screenshots of the mobile screen and testing those.

Thanks!

**Comment by Robdel12** [comment, marked as answer]
Hey @dmaulick! You can upload those using the CLI `upload` command: https://docs.percy.io/docs/cli-upload
